### ISSUE TITLE
LOST FEATURE: cannot move out of tag constraint edit control with keyboard only using Ctrl+Tab

### ISSUE THREAD

**Issue by rajibkhan** [BiglySoftware/BiglyBT#2242]
Earlier (probably in vuze) it was possible to leave / move out of edit control of tag constraint using Ctrl+Tab and Ctrl+Shift+Tab to select tab using only keybaord. It was a helpful feature to move forward to the Save/Reset Button once you're done defining the constraint. It doesn't work now. Now, using Tab only adds a tab space as part of text string in the control. 

Please fix this issue or add this feature back.

**Comment by parg** [comment]
fixed next beta

**Comment by rajibkhan** [comment]
Thx
